### PR TITLE
Fix cvv visibility

### LINKED ISSUE #16: Bug no verso do cartão
ao clicar no flip do cartão, durante a animação é possível ver o cvv e a tarja, mas quando a animação é concluída eles somem. 

OS: MacOS High Sierra 10.13 
Browser: Google Chrome 61.0.3163.100

screen shot 2017-11-06 at 18 07 25

### PR DESCRIPTION
**Pull request by vcapretz**
closes #16

### CODE REVIEW

File: src/assets/css/card.scss [hunk 1]
```diff
@@ -58,31 +63,22 @@ $textShadow: 1px 1px 1px rgba(0, 0, 0, 0.7);
 
       .number {
         position: absolute;
-        font-family: $textFont;
         font-size: 1.15em;
```
Comment (felquis) [inline src/assets/css/card.scss]: Prefer pixels

File: src/assets/css/card.scss [hunk 2]
```diff
@@ -58,31 +63,22 @@ $textShadow: 1px 1px 1px rgba(0, 0, 0, 0.7);
...
         font-weight: normal;
-        color: $textColor;
         letter-spacing: 2.5px;
```
Comment (felquis) [inline src/assets/css/card.scss]: Prefer em for letter-spacing

File: src/assets/css/card.scss [hunk 3]
```diff
@@ -1,6 +1,8 @@
-$textColor: #f6d3fd;
-$textFont: "Overpass Mono", monospaced;
-$textShadow: 1px 1px 1px rgba(0, 0, 0, 0.7);
+:root {
+  --cart-text-color: #fff;
+  --cart-text-font: "Overpass Mono", monospaced;
+  --cart-text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.7);
+}
```
Comment (felquis) [inline src/assets/css/card.scss]: Very good, lets remove node-sass from the project.

File: src/assets/css/card.scss [hunk 4]
```diff
@@ -114,6 +110,7 @@ $textShadow: 1px 1px 1px rgba(0, 0, 0, 0.7);
       }
 
       .cvv {
+        -webkit-backface-visibility: hidden;
```
Comment (felquis) [inline src/assets/css/card.scss]: Use it with cssnext to enable auto-prefix.

File: package.json [hunk 5]
```diff
@@ -55,7 +55,8 @@
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom",
     "storybook": "start-storybook -p 6006",
-    "preview": "storybook-to-ghpages"
+    "preview": "storybook-to-ghpages",
+    "lint": "eslint ."
```
Comment (thiagommedeiros) [inline package.json]: Nice!! Add `yarn lint` into `.travis.yml` as well. 
We can add the `eslint-config-pagarme-react`, what you think about?

See this: https://github.com/mundipagg/mercurio-checkout/commit/ad0d7e60fa591a90c18a4de8c0b4b96ea4febe04

File: README.md [hunk 6]
```diff
@@ -7,7 +7,7 @@
   <a href="https://travis-ci.org/pagarme/react-payment-card-component" target="_blank">
     <img src="https://travis-ci.org/pagarme/react-payment-card-component.svg?branch=master">
   </a>
-  <a href="https://github.com/thiagommedeiros/bus-promise/blob/master/LICENSE" target="_blank">
+  <a href="https://github.com/pagarme/react-payment-card-component/blob/master/LICENSE" target="_blank">
```
Comment (thiagommedeiros) [inline README.md]: Thanks for that!!! lol

File: package.json (hunk 5 above)
Comment (vcapretz) [reply package.json]: I think @augusto-jm-amaral will add both eslint and style lint in another issue, right? 

I'll add the command to travis.yml tho

Comment (augusto-jm-amaral) [reply package.json]: I thought of something like `"lint": "eslint './src/**/*.js' && stylelint './src/**/*.css'"`, but I'm already creating a PR with it.

**Pull request merged**